control-plane/api-gateway/common/diff.go

```diff
@@ -274,8 +274,10 @@ func (e entryComparator) urlRewritesEqual(a, b api.URLRewrite) bool {
 }
 
 func (e entryComparator) retryFiltersEqual(a, b api.RetryFilter) bool {
-	return BothNilOrEqual(a.NumRetries, b.NumRetries) && BothNilOrEqual(a.RetryOnConnectFailure, b.RetryOnConnectFailure) &&
-		slices.Equal(a.RetryOn, b.RetryOn) && slices.Equal(a.RetryOnStatusCodes, b.RetryOnStatusCodes)
+	return a.NumRetries == b.NumRetries &&
```

Comment:
`NumRetries` here is still a pointer so we need the `BothNilOrEqual` right?

Reply:
It's just a `uint32` now

https://github.com/hashicorp/consul/blob/api/v1.26.1/api/config_entry_routes.go#L225